Common/cpp/LayoutAnimations/LayoutAnimationsManager.cpp

```diff
@@ -16,6 +17,20 @@ void LayoutAnimationsManager::configureAnimation(
     exitingAnimations_[tag] = config;
   } else if (type == "layout") {
     layoutAnimations_[tag] = config;
+  } else if (type == "sharedElementTransition") {
+    sharedTransitionAnimations_[tag] = config;
+    if (sharedTransitionGroups_.find(sharedTransitionTag) ==
```

Comment:
use `contains` instead `find` + `end` as it more clearly describes the intent.

Reply:
Isn't that available only since C++20?
https://en.cppreference.com/w/cpp/container/map/contains

Reply:
Also, in this case:
https://en.cppreference.com/w/cpp/container/map/insert
"Inserts element(s) into the container, if the container doesn't already contain an element with an equivalent key."